rust/scx_utils/src/lib.rs

```diff
@@ -36,6 +36,7 @@ mod bpf_builder;
 pub use bpf_builder::BpfBuilder;
 
 pub mod ravg;
+pub mod libbpf_logger;
```

Comment:
Would it make sense to rename `init_loggin()` to something like `init_bpf_logging()` and publish from here so that the users don't have to dig into the sub-module?